pandas/_libs/hashtable.pxd

```diff
@@ -38,7 +38,7 @@ cdef class MultiIndexHashTable(HashTable):
 
     cpdef get_item(self, object val)
     cpdef set_item(self, object key, Py_ssize_t val)
-    cdef inline void _check_for_collision(self, Py_ssize_t loc, object label)
```

Comment:
Is it correct that `_check_for_collisions` is only used in `hashtable.pyx` itself? (or in the included template files) 
If so, I think we can just remove it here from the pxd file altogether (it is already defined as inline in hashtable itself)